tapdb/universe_forest.go

```diff
@@ -68,54 +68,60 @@ func NewBaseUniverseForest(db BatchedUniverseForest) *BaseUniverseForest {
 
 // RootNodes returns the complete set of known base universe root nodes for the
 // set of base universes tracked in the universe forest.
-func (b *BaseUniverseForest) RootNodes(ctx context.Context) ([]universe.BaseRoot, error) {
-	var uniRoots []universe.BaseRoot
+func (b *BaseUniverseForest) RootNodes(
+	ctx context.Context) ([]universe.BaseRoot, error) {
 
-	readTx := NewBaseUniverseForestReadTx()
+	var (
+		uniRoots []universe.BaseRoot
+		readTx   = NewBaseUniverseForestReadTx()
+	)
 
-	dbErr := b.db.ExecTx(ctx, &readTx, func(db BaseUniverseForestStore) error {
-		dbRoots, err := db.UniverseRoots(ctx)
-		if err != nil {
-			return err
-		}
+	dbErr := b.db.ExecTx(
```

Comment:
Not sure if this is a fix as introduces another layer of indentation?

Reply:
`dbErr := b.db.ExecTx(...` was over the line length limit. But I've reverted to save on indentation :+1: